scripts/jiva-tools/jiva_preupgrade_checks.sh

```diff
@@ -36,24 +36,24 @@ function createFragOut() {
 	fi
 }
 
-echo "volume.meta"
+#echo "volume.meta"
 #cat volume.meta
 next=$(getAttr volume "Parent" 5)
 head=$(getAttr volume "Head" 2)
-du -s $head
-size=`du -s $head | awk '{print $1}'`
+#du -s $head
+size=$(du -s "$head" | awk '{print $1}')
 chainLen=1
-echo "$head"
+#echo "$head"
 createFragOut "$head" $chainLen
 
 while [ "$next" != "" ]
 do
 	chainLen=$((chainLen + 1))
 	createFragOut "$next" $chainLen
-	echo "$next.meta"
-	du -s $next
-	cat "$next.meta"
-	img_size=`du -s $next | awk '{print $1}'`
+	#echo "$next.meta"
+	#du -s "$next"
+	#cat "$next.meta"
```

Comment:
If this information is useful for further debugging. This can be printed to a "debug.out" file.

Reply:
Yes, better  to have all these things in debug output. If there is mismatch in chainLen, user can directly post this debug output to us.